### PR TITLE
Test scenario and required page objects

### PR DESCRIPTION
**Pull request by rajmundex**
Jira: [CP-1322]

**Description**

E2E tests environment setup and execution guide:
https://appnroll.atlassian.net/wiki/spaces/CP/pages/1678737459/E2E+test+environment+setup+execution+guide

to execute the test scenario: run Terminal then navigate to ../MuditaCenterRepoFolder/packages/e2e/ folder and execute command:
npx wdio run wdio.conf.ts --spec ./src/specs/basic-in-app-navigation.e2e.ts

[CP-1322]: https://appnroll.atlassian.net/browse/CP-1322?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by igorbezsmertnyi — DISMISSED** [review]

File: packages/e2e/src/specs/basic-in-app-navigation.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+describe("News screen check", () => {
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 10000)
+    })
+  })
+
+  it("Should click on 'Mudita News' tab and check news will load", async () => {
```
[the remark this answers is not in the document]
Comment (rajmundex) [reply packages/e2e/src/specs/basic-in-app-navigation.e2e.ts:16]: Much depends on how tests are planning to be run. Such split is done for selectors which is more important. If such split is added then there will be more work with configuration - currently to create a suite I would add one file path in wdio.conf vs adding multiple filepths to do the same with such split approach. This is topic for some wider discussion. there are pros and cons for both approaches..

**Review by dkarski — COMMENTED** [review]
please add `basic-in-app-navigation.e2e.ts` to `wdio.conf` to allow triggering this e2e on the CI.

**Comment by wojcik92michal** [comment]
> please add `basic-in-app-navigation.e2e.ts` to `wdio.conf` to allow triggering this e2e on the CI.

Maybe there is a particular reason, but couldn't we just add a wildcard to `wdio.conf` (e.g. `./src/specs/**/*.e2e.ts`)?

**Comment by dkarski** [comment]
> @dkarski Maybe there is a particular reason, but couldn't we just add a wildcard to `wdio.conf` (e.g. `./src/specs/**/*.e2e.ts`)?

good point! Please let's do it like in this suggestion 🙏

File: packages/e2e/src/page-objects/news.page.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { ChainablePromiseElement } from "webdriverio"
+import Page from "./page"
+
+class NewsPage extends Page {
```
[the remark this answers is not in the document]
Comment (rajmundex) [reply packages/e2e/src/page-objects/news.page.ts:4]: https://github.com/mudita/mudita-center/pull/756/commits/65af7437ac97a9bd61c8fdba0d2fd3e99f7e26a8

**Comment by rajmundex** [comment]
I spoke with Igor, wildcards is not good option. it does not allow to control the order of tests execution.

File: packages/e2e/src/specs/basic-in-app-navigation.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    const moreNews = await NewsPage.moreNewsButton
+    await expect(moreNews).toBeDisplayed()
+  })
+})
+
+describe("Contacts screen check", () => {
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 5000)
```
[the remark this answers is not in the document]
Comment (rajmundex) [reply packages/e2e/src/specs/basic-in-app-navigation.e2e.ts]: timeouts removed, it was basically to pause execution awaiting app to load, but it does seem to work without them
https://github.com/mudita/mudita-center/pull/756/commits/30816fb7d907dc6804f1b84a2ad10edf25097bfb

File: packages/e2e/src/specs/basic-in-app-navigation.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    })
+  })
+  it("Should click Contacts tab and check Import button is displayed", async () => {
+    const contactsTab = await NavigationTabs.contactsTab
+    //await contactsTab.waitForDisplayed()
```
[the remark this answers is not in the document]
Comment (rajmundex) [reply packages/e2e/src/specs/basic-in-app-navigation.e2e.ts]: https://github.com/mudita/mudita-center/pull/756/commits/30816fb7d907dc6804f1b84a2ad10edf25097bfb

File: packages/e2e/src/specs/basic-in-app-navigation.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+})
+
+describe("Contacts screen check", () => {
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 5000)
+    })
+  })
+  it("Should click Contacts tab and check Import button is displayed", async () => {
```
[the remark this answers is not in the document]
Comment (rajmundex) [reply packages/e2e/src/specs/basic-in-app-navigation.e2e.ts]: for me this gives information what the test is about to do, it is not only what is expected result

**Review by wojcik92michal — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**